featurebyte/models/entity.py

```diff
@@ -83,6 +83,7 @@ class EntityModel(EntityRelationship):
         ID of table with primary key columns associated to the entity
     """
 
+    dtype: Optional[DBVarType] = None
```

Comment:
Curious what is the difference between declaring this as None directly vs `Field(default=None)`?

Reply:
`Field(default=None)` is for the old pydantic version, using `None` is supported & suggested for the new pydantic.